org.sil.scripture-app-builder.yml

```diff
@@ -0,0 +1,114 @@
+# REFERENCES:
+  # Python2.7:  https://github.com/flathub/shared-modules/blob/cc50bdbb34cc3d29bf61cfea5e27cd980b7bda04/python2.7/python-2.7.json
+  # OpenJDK17: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17/blob/branch/23.08/org.freedesktop.Sdk.Extension.openjdk17.yaml
+  # Node20:    https://github.com/flathub/org.freedesktop.Sdk.Extension.node20/blob/branch/23.08/org.freedesktop.Sdk.Extension.node20.yaml
+# FIRST-TIME-BUILD: ~30m on 10-20Mb/s download speed
+app-id: org.sil.scripture-app-builder
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+  # - org.freedesktop.Sdk.Extension.openjdk21
+finish-args:
+  # Window x11 socket access
+  - --socket=x11
+  # Audio access
+  - --socket=pulseaudio
+  # Unsure if the program absolutely needs ssh
+  # - --socket=ssh-auth
+  # Or the printer socket
+  # - --socket=cups
+  # Network access
+  - --share=network
+  # XShm access
+  - --share=ipc
+  # All external devices (catch all for devices we interact with and graphical acceleration)
+  - --device=all
+  # Local file browsing access
+  - --filesystem=host
+  # Temporary file access
+  - --filesystem=/tmp
+  # Filesystem mounts
+  - --filesystem=/mnt
+  # USB Drives
+  - --system-talk-name=org.freedesktop.Udisks2
+  # File saving access
+  - --system-talk-name=org.freedesktop.FileManager1
+
+# Build args give resource access during the build
+build-options:
+  build-args:
+    - --share=network
+
+# App's run command
+command: sab-run.sh
+
+modules:
+
+# For in-app terminal
+- xterm.yml
+
+- python3-aeneas.yml
+
+# For python scripts
+- python3-requests.yaml
+
+# For pwa
+- sab-node20.yml
+
+- openjdk17.yml
+
+- name: scripture-app-builder
+  buildsystem: simple
+  build-commands:
+    - pwd
+    # Source tarball from a docker image and extract to flatpak app
+    - cd /tmp
+    - python3 /run/build/scripture-app-builder/docker_pull.py ghcr.io/sillsdev/scripture-app-builder:12.1
+    - python3 /run/build/scripture-app-builder/extract_layer.py /app/output
+    - rm sillsdev_scripture-app-builder.tar
+    - cd /run/build/scripture-app-builder
```

Comment:
this should already be the current directory. If you get out of part of these steps then it's too complicated.

Reply:
When I switch from docker file to tarball, this should go away.

Reply:
Removed